examples/f16/components/fgbase.py

```diff
@@ -0,0 +1,218 @@
+import threading
+import socket
+import time
+import math
+import typing
+
+import numpy as np
+import pymap3d as pm
+
+from abc import ABC
+from scipy.spatial.transform import Rotation
+
+import csaf.trace
+
+class Dubins2DConverter():
+    """
+    Originally from John McCarroll, modified by Michal Podhradsky
+
+    Converts orientation and rotation from ENU to ECEF
+    """
+
+    @classmethod
```

Comment:
Should this be a `staticmethod`? `cls` is not used

Reply:
Fixed